src/data/multiset/basic.lean

```diff
@@ -848,6 +848,22 @@ begin
     simp }
 end
 
+lemma map_eq_cons' [decidable_eq α] (f : α → β) {s : multiset α} {t : multiset β} (b : β) :
```

Comment:
Why the `'` in the name? I don't see any `map_eq_cons`.

Also, what about making `s` and `t` explicit?